sdk/servicebus/service-bus/test/public/utils/envVarUtils.ts

```diff
@@ -1,8 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-export const isNode =
-  !!process && !!process.version && !!process.versions && !!process.versions.node;
+import { isNode } from "./isNode";
```

Comment:
nit: this could be imported from `@azure/core-util`.

Reply:
yes, since we only use it in test. Fixed